### PR TITLE
upgrade to CKEditor version 4.14.1

### PR DESCRIPTION
**Pull request by jrief**
This is just a binary upgrade to CKEditor version 4.14.1.
In addition `gulp build`now runs in NodeJS version 12.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 
 REQUIREMENTS = [
+    'Django<3.1',
```
Comment (FinalAngel) [inline setup.py:9]: please don't add a django restriction here

**Review by FinalAngel — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (yakky) [reply setup.py:9]: maybe we could just wait for #543 to pass and then rebase this

Comment (jrief) [reply setup.py:9]: This was a test, because Travis-CI failed for reasons I can not retrace.

Comment (FinalAngel) [reply setup.py]: 👍

Comment (yakky) [reply setup.py]: :+1:

**Review by FinalAngel — APPROVED** [review]

**Comment by FinalAngel** [comment]
@jrief shall we merge this as well for the next release?

**Comment by jrief** [comment]
> shall we merge this as well for the next release?

@FinalAngel Yes, and I will try to always keep up to date with the latest version of CKEditor from the 4 series.

**Comment by FinalAngel** [comment]
@jrief 👍 , can you resolve the conflicts? I'll prepare a release right after this gets merged.

**Comment by FinalAngel** [comment]
awesome, thank you @jrief

**Pull request merged**